metricbeat/module/prometheus/query/_meta/docs.asciidoc

```diff
@@ -0,0 +1,54 @@
+This is the `query` metricset to query from querring API[https://prometheus.io/docs/prometheus/latest/querying/api/#expression-queries] of Promtheus.
+
+
+[float]
+=== Configuration
+
+[float]
+==== Instant queries
+
+[source,yaml]
```

Comment:
Avoid "Title to block" if possible. Present, in natural language, what the next piece of code is. For example, don't write here now "An example of instant queries" because that will be redundant with the title. Instead, something like "An example config of the sum of the HTTP requests rate in a window of 1 minute"